pkg/storaged/fsys-tab.jsx

```diff
@@ -381,20 +410,25 @@ export function mounting_dialog(client, block, mode, forced_options) {
             opts.push("ro");
         if (opt_never_auto)
             opts.push("x-cockpit-never-auto");
+        if (opt_nofail)
+            opts.push("nofail");
+        if (opt_netdev)
+            opts.push("_netdev");
         if (forced_options)
             opts = opts.concat(forced_options);
         if (extra_options)
             opts = opts.concat(extra_options);
-        return maybe_set_crypto_options(null, false).then(() => maybe_update_config(old_dir, unparse_options(opts)));
+        return (maybe_set_crypto_options(null, false, null, null)
+                .then(() => maybe_update_config(old_dir, unparse_options(opts))));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221115-130151-a219e579-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)